openwisp_radius/tests/test_api.py

```diff
@@ -2414,3 +2414,64 @@ def test_change_phone_number_restriction(self):
             self.assertEqual(user.phone_number, phone_number)
 
         app_settings.ALLOWED_MOBILE_PREFIXES = []
+
+
+class TestApiRejectPhoneToken(ApiTokenMixin, BaseTestCase):
+
+    _test_email = 'test@openwisp.org'
+
+    def setUp(self):
+        super().setUp()
+        radius_settings = self.default_org.radius_settings
+        radius_settings.sms_verification = False
+        radius_settings.sms_sender = '+595972157632'
+        radius_settings.sms_meta_data = {
```

Comment:
avoid setting the sms related fields, this will give us a more realistic test env